### PR TITLE
Add support for Avalonia 0.11.

### PR DESCRIPTION
**Pull request by laolarou726**
This change added support for Avalonia 0.11-beta-001.

### CODE REVIEW

File: AsyncImageLoader.Avalonia.Demo/App.axaml [hunk 1]
```diff
@@ -7,7 +7,6 @@
     </Application.DataTemplates>
 
     <Application.Styles>
-        <FluentTheme Mode="Light"/>
```
Comment (SKProCH) [inline AsyncImageLoader.Avalonia.Demo/App.axaml:10]: Did you start demo app? I sure what nothing will be rendered without styles

File: AsyncImageLoader.Avalonia.Demo/Pages/AdvancedImagePage.axaml [hunk 2]
```diff
@@ -8,15 +8,17 @@
              x:Class="AsyncImageLoader.Avalonia.Demo.Pages.AdvancedImagePage">
   <Grid ColumnDefinitions="* 8 150 150" RowDefinitions="Auto Auto Auto Auto"
         HorizontalAlignment="Center">
-    <TextBlock Grid.Column="2" Grid.Row="0" HorizontalAlignment="Center">AdvancedImage</TextBlock>
-    <TextBlock Grid.Column="3" Grid.Row="0" HorizontalAlignment="Center">Image</TextBlock>
+    <TextBlock Grid.Column="2" Grid.Row="0" HorizontalAlignment="Center" Text="AdvancedImage"/>
+    <TextBlock Grid.Column="3" Grid.Row="0" HorizontalAlignment="Center" Text="Image"/>
```
Comment (SKProCH) [inline AsyncImageLoader.Avalonia.Demo/Pages/AdvancedImagePage.axaml:12]: I thing what moving controls content is an unnecessary change, not related to compatibility with avalonia 11

File: AsyncImageLoader.Avalonia/AdvancedImage.axaml.cs [hunk 3]
```diff
@@ -69,7 +69,7 @@ public class AdvancedImage : ContentControl {
 
             var loaderChangedObservable = LoaderProperty.Changed
                 .Where(args => args.IsEffectiveValueChange)
-                .Where((args, i) => i == 0 || args.Sender.GetValue(ShouldLoaderChangeTriggerUpdateProperty))
+                .Where((args, i) => i == 0 || (bool)args.Sender.GetValue(ShouldLoaderChangeTriggerUpdateProperty))
```
Comment (SKProCH) [inline AsyncImageLoader.Avalonia/AdvancedImage.axaml.cs:72]: This change will trigger possible null reference exception warning. We should suppress it via adding `!` to the end of expression

**Review by SKProCH — CHANGES_REQUESTED** [review]
Also, please remove temp files in `v17` and `FileContentIndex` folders - they do not belong to the source code

**Comment by SKProCH** [comment]
I can implement it by myself tomorrow, if you want

File: AsyncImageLoader.Avalonia.Demo/Pages/AdvancedImagePage.axaml (hunk 2 above)
Comment (laolarou726) [reply AsyncImageLoader.Avalonia.Demo/Pages/AdvancedImagePage.axaml:12]: In the latest Avalonia 0.11 version. They are not going to support putting text like

```xaml
<TextBlock>xxx</TextBlock> # This will throw the error
```

As a result, we have to put all the texts in the Text property.

**Comment by laolarou726** [comment]
I just did a quick fix for those issues. Everything should be fine right now!

**Review by SKProCH — APPROVED** [review]
Looks fine, thanks

**Comment by SKProCH** [comment]
Temporarily, I will merge them in a separate branch, and post the Avalonia 11 package myself (in a short time)

**Comment by SKProCH** [comment]
Hello, @laolarou726 
I published this as `v3.0.0-avalonia11` at nuget
